psh/test-exit.py

```diff
@@ -0,0 +1,58 @@
+# Phoenix-RTOS
+#
+# phoenix-rtos-tests
+#
+# simple psh exit command test
+#
+# Copyright 2022 Phoenix Systems
+# Author: Damian Loewnau
+#
+# This file is part of Phoenix-RTOS.
+#
+# %LICENSE%
+#
+
+import psh.tools.psh as psh
+from psh.tools.proc import get_process_list, create_psh_processes
+
+
+def _count_sleep_pshs(p):
+    return sum((proc['state'] == 'sleep' and proc['task'] in ('psh', '/bin/psh')) for proc in get_process_list(p))
+
+
+def assert_one_proc(p, already_spawned):
+    psh.assert_exec(p, 'psh', msg="Prompt hasn't been seen when creating new psh process using runfile")
+    assert _count_sleep_pshs(p) == already_spawned + 1, "New psh process not seen after launching psh by runfile!"
+    psh.assert_cmd(
+                   p,
+                   'exit',
+                   result='success',
+                   msg="Prompt hasn't been seen after first exit (two psh processes should be run)")
+
+    assert _count_sleep_pshs(p) == already_spawned, "".join(
+        "The count of sleepy psh processes haven't been decreased after calling exit!")
+
+
+def assert_multi_procs(p, count, already_spawned):
+
+    create_psh_processes(p, count, already_spawned)
```

Comment:
Reminder: In the newer version of the `test-kill`, the `create_psh_processes` has been modified and no longer has a third argument. However, this does not affect the test's functionality.
```suggestion
    create_psh_processes(p, count)
```